app/src/main/java/com/example/cs65_final_project/activities/RecipeViewActivity.java

```diff
@@ -1,8 +1,9 @@
 package com.example.cs65_final_project.activities;
 
-import androidx.appcompat.app.AppCompatActivity;
-
 import android.os.Bundle;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
 
 import com.example.cs65_final_project.R;
 
```

Comment:
@jamesversch17 Initial Recipe View activity is merged in - this is likely where we'll want to stick a "share" button that populates the feed with Recipes